app/skeleton/skeleton.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"github.com/dedis/cothority/lib/app"
+	"github.com/dedis/cothority/lib/conode"
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/dedis/cothority/lib/monitor"
+	"github.com/dedis/cothority/lib/sign"
+	"time"
+)
+
+// This file is the first draft to a skeleton app where you have all the
+// basics to run your own cothority tree. This include the main where you handle
+// the configuration + the "running" part. It also include a basic Round
```

Comment:
Maybe better?
This includes a main() function which shows how to configure and run a cothority based application.